src/main/java/org/cyclops/everlastingabilities/GeneralConfig.java

```diff
@@ -107,6 +107,28 @@
     @ConfigurableProperty(category = ConfigurableTypeCategory.GENERAL, comment = "The amount of exhaustion that should by applied to the player per active ability per second.", isCommandable = true)
     public static double exhaustionPerAbilityTick = 0.01;
 
+    /**
+     * Mobs that are immune to hostile potion effect abilities
+     */
+    @ConfigurableProperty(category = ConfigurableTypeCategory.GENERAL, comment = "These mobs will not be affected by hostile area potion effects such as poison or weakness.")
```

Comment:
`requiresMcRestart` should be set to `true`, due to how you have implemented config value synchronization.

Alternatively, you could set a `changedCallback` so that this list can be changed at runtime. (EvilCraft does this, if you need an example).

The second option would be nicer, but not required.